### PR TITLE
Qa docker update

### PR DESCRIPTION
**Pull request by regit**
- [x] I have read the contributing guide lines at https://redmine.openinfosecfoundation.org/projects/suricata/wiki/Contributing
- [x] I have signed the Open Information Security Foundation contribution agreement at https://suricata-ids.org/about/contribution-agreement/
- [x] I have updated the user guide (in doc/userguide/) to reflect the changes made (if applicable)

Link to [redmine](https://redmine.openinfosecfoundation.org/projects/suricata/issues) ticket:

Describe changes:
- update to latest stable buildbot
- add rust support
- add suricata-verify

Note: once merge we need an upgrade of https://redmine.openinfosecfoundation.org/projects/suricata/wiki/SuriDockerQA to document suricata-verify usage (needs to be cloned under qa directory).

[PRScript](https://redmine.openinfosecfoundation.org/projects/suricata/wiki/PRscript) output:
- PR regit: https://buildbot.openinfosecfoundation.org/builders/regit/builds/417
- PR regit-pcap: https://buildbot.openinfosecfoundation.org/builders/regit-pcap/builds/199

### CODE REVIEW

File: qa/docker/buildbot.cfg [hunk 1]
```diff
@@ -126,11 +125,13 @@ pcaps_list = [ os.path.join(PCAP_PATH, pcap) for pcap in pcaps_list if pcap.ends
...
+factory_stress_pcap.addStep(ShellCommand(command=["./configure","--enable-debug-validation", "--enable-rust","ac_cv_func_malloc_0_nonnull=yes","ac_cv_func_realloc_0_nonnull=yes"],env={"CFLAGS" : "-fsanitize=address -fno-omit-frame-pointer"}))
 factory_stress_pcap.addStep(ShellCommand(command=["make"]))
 factory_stress_pcap.addStep(ShellCommand(command=["sudo", "make","install"]))
 factory_stress_pcap.addStep(ShellCommand(command=["sudo", "rm", "-f", "/usr/local/etc/suricata/suricata.yaml"]))
 factory_stress_pcap.addStep(ShellCommand(command=["sudo", "make","install-conf"]))
+if os.path.exists('/data/oisf/qa/suricata-verify/run.py'):
+    factory_stress_pcap.addStep(ShellCommand(command=["/data/oisf/qa/suricata-verify/run.py","--outdir","/tmp"]))
 factory_stress_pcap.addStep(ShellCommand(command=["make","clean"]))
 factory_stress_pcap.addStep(ShellCommand(command=["sudo", "ldconfig"]))
 for pfile in pcaps_list:
```
Comment (victorjulien) [inline qa/docker/buildbot.cfg]: in the dev training we needed src/suricata instead of the absolute path here

File: qa/docker/buildbot.cfg [hunk 2]
```diff
@@ -126,11 +125,13 @@ pcaps_list = [ os.path.join(PCAP_PATH, pcap) for pcap in pcaps_list if pcap.ends
...
 factory_stress_pcap.addStep(ShellCommand(command=["./autogen.sh"]))
-factory_stress_pcap.addStep(ShellCommand(command=["./configure","--enable-debug-validation"],env={"CFLAGS" : "-fsanitize=address -fno-omit-frame-pointer"}))
+factory_stress_pcap.addStep(ShellCommand(command=["./configure","--enable-debug-validation", "--enable-rust","ac_cv_func_malloc_0_nonnull=yes","ac_cv_func_realloc_0_nonnull=yes"],env={"CFLAGS" : "-fsanitize=address -fno-omit-frame-pointer"}))
 factory_stress_pcap.addStep(ShellCommand(command=["make"]))
 factory_stress_pcap.addStep(ShellCommand(command=["sudo", "make","install"]))
 factory_stress_pcap.addStep(ShellCommand(command=["sudo", "rm", "-f", "/usr/local/etc/suricata/suricata.yaml"]))
 factory_stress_pcap.addStep(ShellCommand(command=["sudo", "make","install-conf"]))
+if os.path.exists('/data/oisf/qa/suricata-verify/run.py'):
+    factory_stress_pcap.addStep(ShellCommand(command=["/data/oisf/qa/suricata-verify/run.py","--outdir","/tmp"]))
 factory_stress_pcap.addStep(ShellCommand(command=["make","clean"]))
```
Comment (victorjulien) [inline qa/docker/buildbot.cfg]: this one then also needs to be moved otherwise the src/suricata is gone again

**Comment by victorjulien** [comment]
Needs some fixes.

File: qa/docker/buildbot.cfg (hunk 1 above)
Comment (regit) [reply qa/docker/buildbot.cfg]: I'm going to retest but this version may be correct as the script is running before the `make clean`.

File: qa/docker/buildbot.cfg (hunk 2 above)
Comment (regit) [reply qa/docker/buildbot.cfg]: Tests seem to show it is ok. `make clean` is after the test. I'm rebasing and resubmitting.

**Pull request closed without merging**